sae/blocks.go

```diff
@@ -115,7 +115,7 @@ func (vm *VM) buildBlock(
 	if parentTime := parent.BuildTime(); hdr.Time < parentTime {
 		return nil, fmt.Errorf("%w: %d < %d", errBlockTimeBeforeParent, hdr.Time, parentTime)
 	}
-	if maxTime := uint64(vm.config.Now().Add(maxFutureBlockTime).Unix()); hdr.Time > maxTime { //nolint:gosec // Time won't overflow for quite a while
+	if maxTime := unix(vm.config.Now().Add(maxFutureBlockTime)); hdr.Time > maxTime {
```

Comment:
(fyi) I implemented the `unix()` function specifically to avoid the need for `//nolint:gosec` everywhere. Sorry for not picking it up in my review of your PR.